setup.py

```diff
@@ -55,8 +55,12 @@ def __str__(self):
     author_email="sdpdev+katsdpimager@ska.ac.za",
     url="https://github.com/ska-sa/katsdpimager/",
     packages=find_packages(),
-    package_data={'': ['imager_kernels/*.mako', 'imager_kernels/*/*.mako']},
-    scripts=["scripts/imager.py", "scripts/imager-mkat-pipeline.py", "scripts/fits-video.py"],
+    package_data={'': ['imager_kernels/*.mako', 'imager_kernels/*/*.mako',
+                       'models/*/*/*/*.h5']},
```

Comment:
For the moment a single beam pattern at one elevation for a single antenna is 577Mb at the resolution and extent provided (which is fairly arbitrarily chosen, and lets call it medium resolution). If Bruce wants to use some GPU code to sample the beam then the storage requirement becomes 2.1GB for 64 antennas at one elevation using 'existing techniques'. In the future using techniques I have only partly implemented this storage requirement would become 2.1GB for 64 antennas at all elevations. For meerkat according to its design specs, one technically only need one array average pattern at one elevation. For SKA this may not be good enough. And it is unproven if for MeerKAT it is a practical requirement (billion to one dymanic range) rather than a system engineering requirement.

Reply:
Oh and above is for one band. Another that amount for UHF, and possibly likewise for S band.

Reply:
There is scope for reducing the storage required even more, to almost the size of a single beam for all antennas, but that will not happen until a years time.

Reply:
Where does the billion to 1 dynamic range requirement for MeerKAT come form? That's a tad excessive! Even for SKA!

Reply:
> How large is a single model file at present? I worry that once you move to 2d beams with elevation and polarisation dependence then this will become quite a large file to store in the repository.

At the moment these files are 266 kB for L-band, 262 kB for UHF (no idea why one is slightly smaller - maybe some HDF5 compression). I definitely don't want to be checking in full 2D beam models, and per-antenna models don't belong in the repository because they'll evolve over time. My plan is to put them in S3 and reference them from telstate, but that's still just an idea.